af/firefox-ios.xliff

```diff
@@ -3995,6 +4011,10 @@ Private Tab</source>
         <source>Very dissatisfied</source>
         <note>On the microsurvey, this is the title for one of the options that the user can select to answer the survey.</note>
       </trans-unit>
+      <trans-unit id="Microsurvey.Survey.Options.LikertScaleOption6.v129" xml:space="preserve">
+        <source>I don't use it</source>
```

Comment:
@cyndichin Can you please update the apostrophe so it's a typographic apostrophe?
We should use that in the strings (not necessary in l10n comments)

Reply:
thanks! will update! Is there documentation on this so I can link it to my PR? I will update it to `I don’t use it` let me know if there's anything else I need.

Reply:
PR has been merged.

Reply:
@cyndichin Yes there is documentation [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#land-good-quality-strings) about this.

Generally speaking, this entire documentation is comes in very handy when working with strings: [Localization Best Practices for Developers](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html).

It comes in really handy for engineers who start working with localization files, so feel free to share the love in those cases! 😃 thank you!